packages/terra-image/src/Image.jsx

```diff
@@ -59,12 +69,13 @@ const Image = ({
       alt={alt}
       height={height}
       width={width}
+      onLoad={onLoad}
+      onError={onError}
```

Comment:
Since I'm still new to React development I'm trying to decipher what you are implying by posting the link.  Looking at the code again, I think you are saying that customProps would handle the onLoad and onError events so they aren't needed?  If so, should src, alt, height, and width also be removed?  The description for terra-Image should also be updated to explicitly state it's a wrapper around image instead of stating "The terra-image component provides styling for visual imagery."